blenderproc/python/renderer/SegMapRendererUtility.py

```diff
@@ -300,13 +300,18 @@ def _set_world_background_color(color: mathutils.Vector):
     nodes = bpy.context.scene.world.node_tree.nodes
     links = bpy.context.scene.world.node_tree.links
 
+    background_node = Utility.get_the_one_node_with_type(nodes, "Background")
+
     # Unlink any incoming link that would overwrite the default value
-    if len(nodes.get("Background").inputs['Color'].links) > 0:
-        links.remove(nodes.get("Background").inputs['Color'].links[0])
+    if len(background_node.inputs['Color'].links) > 0:
+        links.remove(background_node.inputs['Color'].links[0])
```

Comment:
Doesn't this only remove the first link? Not `any` links.

Reply:
At the same time, shouldn't we remove all other nodes. To clean up the material, not that there are 1000 nodes, which are all unused.

Reply:
> Doesn't this only remove the first link? Not any links.

There can only be at most one link to an input socket. If a second one is connected, the first one gets automatically disconnected

> At the same time, shouldn't we remove all other nodes. To clean up the material, not that there are 1000 nodes, which are all unused.

As these other nodes would not be connected to the output node, I would hope that blender ignores them anyway and that they are not a big performance issue. Also, I don't think that case is so likely^^